docs/source/en/_toctree.yml

```diff
@@ -1,36 +1,39 @@
-- sections:
+- title: Get started
+  sections:
   - local: index
-    title: 🧨 Diffusers
+    title: Diffusers
+  - local: installation
+    title: Installation
   - local: quicktour
     title: Quicktour
   - local: stable_diffusion
     title: Effective and efficient diffusion
-  - local: installation
-    title: Installation
-  title: Get started
-- sections:
-  - local: tutorials/tutorial_overview
-    title: Overview
-  - local: using-diffusers/write_own_pipeline
-    title: Understanding pipelines, models and schedulers
-  - local: tutorials/autopipeline
-    title: AutoPipeline
-  - local: tutorials/basic_training
-    title: Train a diffusion model
-  title: Tutorials
-- sections:
+
+- title: Pipeline
```

Comment:
Should we call it `DiffusionPipeline`?

I think we should really take the step to make people understand the beauty behind abstracting the core logic of how different components are connected. Could be a good opportunity in this PR to do so, if that's not already evident enough. WDYT? I created the following GIF for a presentation of mine which could be reused.

![generation_pipeline](https://github.com/user-attachments/assets/7575c0ce-4b40-4168-86ea-a32aa497f75e)

Reply:
Sounds good to me!